### DISCUSSION TITLE
llama-7b-instruct-32k.ggml

### DISCUSSION THREAD

**Discussion by TeaCult** [oobabooga/text-generation-webui#4016] [Q&A]
trying to run together ai' s trained 32k 7b model. But I cannot achive satisfactory results. I am using --n_ctx=32k  and config.json states rope scaling factor should be 8 is it the linear compression ? How can I set rope scaling factor for llama.cpp 

Because if I set compression to 8 it kinda works but goes out of the text. Otherwise it babbles meaninglessly. 

Could you instruct us how to use 7b 32k ggml model. Thank you.

**Comment by philtimmes** [comment]
GGML models are not supported anymore. You need to use GPTQ or GGUF models.

**Reply by TeaCult** [reply]
Ok. What would be the parameters to use llama-2-7b-32k-Instruct.gguf ? 
Thx
